sphinx_gherkindoc/writer.py

```diff
@@ -359,6 +362,11 @@ def table(table: behave.model.Table, inline: bool = False) -> None:
             f" options are: {list(parsers.keys())}"
         )
     feature = feature_class(root_path, source_path)
+
+    included_scenarios = get_all_included_scenarios(feature, include_tags, exclude_tags)
+    if not included_scenarios:
+        return None
```

Comment:
I was concerned about this at first, but I see here that all scenarios are always returned if the tags are not passed:
https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/pull/40/files?file-filters%5B%5D=.py#diff-3d557193ea331bdbcac3270cc1326aafR293